samples/bookinfo/src/mongodb/Dockerfile

```diff
@@ -12,7 +12,7 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-FROM mongo
+FROM mongo:4.0
 RUN mkdir -p /app/data/
 COPY ratings_data.json /app/data/
```

Comment:
Instead of `Run mkdir`, we should be able to replace it with `WORKDIR /app/data` followed by `COPY ratings_data.json .`